tests/test_visitors/test_ast/test_general/test_wrong_keyword/test_comprehensions.py

```diff
@@ -92,13 +96,21 @@
     set_ifs_single,
     set_without_ifs,
 ])
+@pytest.mark.parametrize('is_async', [True, False])
 def test_if_keyword_in_comprehension(
     assert_errors,
     parse_ast_tree,
     code,
+    is_async,
     default_options,
 ):
     """Testing that using `if` keyword is allowed."""
+    if is_async:
+        code = code.format(async_stmt='async ')
+        code = code.lstrip()  # remove heading \n symbol
```

Comment:
Why do you think that this line is required?
We are already using `dedent` here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/conftest.py#L33

Or is there any specific problem that I can not see?

Reply:
`dedent` doesn't remove empty lines.
Example
```python
>>> from textwrap import dedent
>>> example = """
... node = ...
... """
>>> dedent(example)
'\nnode = ...\n'
```

If you want to avoid empty line there should be backslash. 
Example from [docs](https://docs.python.org/3.7/library/textwrap.html#textwrap.dedent)
```python
def test():
    # end first line with \ to avoid the empty line!
    s = '''\
    hello
      world
    '''
    print(repr(s))          # prints '    hello\n      world\n    '
    print(repr(dedent(s)))  # prints 'hello\n  world\n'
```

Reply:
Why should we care about the new lines? Does it break something?

Reply:
Yeah.
There are two new lines after `def`
```python
async_wrapper.format(comprehension=dedent(list_ifs_multiple))
"\nasync def f():\n    \nnodes = [\n    node\n    {async_stmt}for node in 'abc'\n    if node != 'a' if node != 'b' if node != 'c'\n]\n\n"
```

Reply:
Also for functions we need proper indentations. And all options that I tried were ugly.